installers/win_installer.py

```diff
@@ -0,0 +1,359 @@
+
+    Returns the path to the newly created environment's Python executable.
+    """
+    fullpath = os.path.join(target_directory, name)
+    if conda_path:
+        command = [
+            conda_path, "create",
+            "-p", os.path.normpath(fullpath),
+            "python={}".format(python_version),
+            "-y"]
+        env_path = os.path.join(fullpath, "python.exe")
+    else:
+        command = [sys.executable, "-m", "venv", fullpath]
+        env_path = os.path.join(fullpath, "Scripts", "python.exe")
+    subprocess_run(command)
+    return env_path
+
+
+def pip_freeze(python, encoding):
+    """
+    Return the "pip freeze --all" output as a list of strings.
+    """
+    print("Getting frozen requirements.")
+    output = subprocess.check_output([python, "-m", "pip", "freeze", "--all"])
+    text = output.decode(encoding)
+    return text.splitlines()
+
+
+def about_dict(repo_root, package):
+    """
+    Return the package about dict.
+
+    keys are the __variables__ in <package>/__init__.py.
+    """
+    DUNDER_ASSIGN_RE = re.compile(r"""^__\w+__\s*=\s*['"].+['"]$""")
+    about = {}
+    with open(os.path.join(repo_root, package, "__init__.py")) as f:
+        for line in f:
+            if DUNDER_ASSIGN_RE.search(line):
+                exec(line, about)
+    return about
+
+
+def pypi_wheels_in(requirements, skip):
+    """
+    Return a list of the entries in requirements (distributed as wheels).
+
+    Where requirements is a list of strings formatted like "name==version".
+    """
+    print("Checking for wheel availability at PyPI.")
+    wheels = []
+    for requirement in requirements:
+        name, _, version = requirement.partition("==")
+        # Needed to detect the package being installed from source
+        # <package> @ <path to package>==<version>
+        name = name.split('@')[0].strip()
+        if name in skip:
+            print("-", requirement, "skipped")
+        else:
+            print("-", requirement, end=" ")
+            package = yarg.get(name)
+            releases = package.release(version)
+            if not releases:
+                raise RuntimeError(
+                    "ABORTING: Did not find {!r} at PyPI. "
+                    "(bad meta-data?)".format(
+                        requirement
+                    )
+                )
+            if any(r.package_type == "wheel" for r in releases):
+                wheels.append(requirement)
+                feedback = "ok"
+            else:
+                feedback = "missing"
+            print(feedback)
+    return wheels
+
+
+def package_name(requirement):
+    """
+    Return the name component of a `name==version` formated requirement.
+    """
+    requirement_name = requirement.partition("==")[0].split("@")[0].strip()
+    return requirement_name
+
+
+def packages_from(requirements, wheels, skip_packages):
+    """
+    Return a list of the entires in requirements that aren't found in wheels.
```

Comment:
```suggestion
    Return a list of the entries in requirements that aren't found in wheels.
```

Minor spelling error.